### ISSUE TITLE
localize the string based messages

### ISSUE THREAD

**Issue by engblh** [kazupon/vue-i18n#530]
I know this crazy question, but just curious 

From backend i receive the message like - "Record updated"

can i localized it ?

```
{
"messages":{
"Record updated":"localized string"
}
}
```

and then use it like
```
$t('messages.Record updated')
```

### vue & vue-i18n version
ex: 2.1.0, 5.0

**Comment by exoego** [comment]
Yes, using message key like `"Record updated"` will be eventually possible after new release, since message key can contain spaces (fixed in PR #533 but not released yet).

**Comment by engblh** [comment]
@kazupon great job team. With latest release, this feature is supported. Thanks to the team. You can close the issue
